lms/static/js/spec/student_account/hinted_login_spec.js

```diff
@@ -0,0 +1,71 @@
+define([
+    'jquery',
+    'underscore',
+    'common/js/spec_helpers/template_helpers',
+    'common/js/spec_helpers/ajax_helpers',
+    'js/student_account/views/HintedLoginView',
+], function($, _, TemplateHelpers, AjaxHelpers, HintedLoginView) {
+    'use strict';
+    describe('edx.student.account.HintedLoginView', function() {
+
+        var view = null,
+            requests = null,
+            PLATFORM_NAME = 'edX',
+            THIRD_PARTY_AUTH = {
+                currentProvider: null,
+                providers: [
+                    {
+                        id: 'oa2-google-oauth2',
+                        name: 'Google',
+                        iconClass: 'fa-google-plus',
+                        loginUrl: '/auth/login/google-oauth2/?auth_entry=account_login',
+                        registerUrl: '/auth/login/google-oauth2/?auth_entry=account_register'
+                    },
+                    {
+                        id: 'oa2-facebook',
+                        name: 'Facebook',
+                        iconClass: 'fa-facebook',
+                        loginUrl: '/auth/login/facebook/?auth_entry=account_login',
+                        registerUrl: '/auth/login/facebook/?auth_entry=account_register'
+                    }
+                ]
+            },
+            HINTED_PROVIDER = "oa2-google-oauth2";
+
+        var createHintedLoginView = function(test) {
+            // Initialize the login view
+            view = new HintedLoginView({
+                thirdPartyAuth: THIRD_PARTY_AUTH,
+                hintedProvider: HINTED_PROVIDER,
+                platformName: PLATFORM_NAME
+            });
+
+            // Mock the redirect call
+            spyOn( view, 'redirect' ).andCallFake( function() {} );
```

Comment:
True; I had just copy-pasted that line from `account_spec`. If I need to amend this commit I'll fix that up :)